src/SwedbankPay/Api/Service/Paymentorder/Request/Test.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace SwedbankPay\Api\Service\Paymentorder\Request;
+
+use SwedbankPay\Api\Service\Base\Request\Test as BaseTest;
+use SwedbankPay\Api\Service\Paymentorder\Resource\PaymentorderObject;
+use SwedbankPay\Api\Service\Paymentorder\Resource\PaymentorderPayeeInfo;
+use SwedbankPay\Api\Service\Paymentorder\Resource\Request\Paymentorder;
+use SwedbankPay\Api\Client\Exception;
+
+class Test extends BaseTest
+{
+    /**
+     * Test constructor.
+     * @param string $merchantToken
```

Comment:
Please change all occurrences of `merchantToken` with `accessToken` (see WOO-77 in Jira).

```suggestion
     * @param string $accessToken
```